server.js

```diff
@@ -536,6 +536,14 @@ app.post('/submit-network', asyncMiddleware( async (req, res, next) => {
             network = networkbuffer.toString();
             hash = checksum(network, 'sha256');
 
+            // Start parsing network weights
+            var weights = network.split("\n"); 
```

Comment:
Would it be better to count the number of newlines directly in network instead of doing the split()? Since the network string might be really large, we'd double the ram usage of a network upload by splitting it into the weights array. Note a 20x256 network is around 256 meg uncompressed, is my concern.

Reply:
Did some searching, here are some approaches. They warn against split(). https://gist.github.com/lsauer/2757250